src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37229/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37229/project/issues?id=sample&pullRequest=5609&issues=AWcsYSXNt1KJ9A5M-0-o&open=AWcsYSXNt1KJ9A5M-0-o)

[ISSUE_KEY]: AWcsYSXNt1KJ9A5M-0-o